docs/commands/mirror-images.md

```diff
@@ -17,8 +17,8 @@ Usage
 
 ### Source Registry Entitlements
 - `--ibm-entitlement IBM_ENTITLEMENT_KEY` IBM Entitlement Key
-- `--redhat-username REDHAT_CONNECT_USERNAME` Red Hat Connect Username (only required when mirroring UDS images)
-- `--redhat-password REDHAT_CONNECT_PASSWORD` Red Hat Connect Password (only required when mirroring UDS images)
+- `--redhat-username REDHAT_CONNECT_USERNAME` Red Hat Connect Username (only required when mirroring UDS/DRO images)
+- `--redhat-password REDHAT_CONNECT_PASSWORD` Red Hat Connect Password (only required when mirroring UDS/DRO images)
```

Comment:
The need to mirror images from redhat registry here was one of the problems with UDS, all content for DRO should be coming from the IBM entitled registry, is that not the case?

Reply:
will have to update the ansible for pulling dro registry as a part of dependencies, will handle it in separate PR